ENG4000-Team-A/capstone-project
# Integrated MachineCard, updated Machine model, added Machine view

Added new Machine attribute machineType.
Added get views for Machine objects. /machines/ will return all machines. /machines/?id=1 will return machine with id 1.
Integrated machine overview & machine card page with backend.

To test:
1. run django server
2. run react app
3. add machine objects using admin page.
4. go to localhost:3000/ , you should see the machines
